### ISSUE TITLE
Add instruction on how to add options to `select` widget

### ISSUE THREAD

**Issue by zanona** [netlify/netlify-cms#764]
At the moment https://www.netlifycms.org/docs/widgets/ doesn't cover how to add options to the `select` widget.

It would be good to have this documented and also mentioned about the ability to set distinct `labels` and `values`

```yaml
collections:
  foo:
    - {label: 'Option list', name: option-list', widget: select, options: [one, two,three]
    - {label: 'Option list', name: option-list', widget: select, options: [{label: 'Web Development', value: webdev]
```

https://github.com/netlify/netlify-cms/blob/master/src/components/Widgets/SelectControl.js#L18-L23

**Comment by verythorough** [comment]
This would be great to add to the Widget docs re-vamp in #619

**Comment by verythorough** [comment]
Not quite closed yet. #806 addresses part of the issue, but we still need to add info on labels and values.

**Comment by verythorough** [comment]
Closed by #876
